vmm/src/vm_config.rs

```diff
@@ -462,6 +462,11 @@ pub fn default_consoleconfig_file() -> Option<PathBuf> {
     None
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+pub struct DebugconConfig {
+    pub file: PathBuf,
```

Comment:
So you don't support printing to stdout - that seems a limitation. Perhaps you could reuse `ConsoleOutputMode` or even `ConsoleConfig`.

Reply:
My intention was to get early feedback. I'm very open for more options!